composer.json

```diff
@@ -16,16 +16,33 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.9",
+        "symfony/console": "~3.3|~4.0",
+        "symfony/config": "~3.3|~4.0",
+        "symfony/dependency-injection": "~3.3|~4.0",
+        "symfony/framework-bundle": "~3.3|~4.0",
+        "symfony/http-kernel": "~3.3|~4.0",
+        "symfony/polyfill-php70": "~1.0",
         "symfony/security-acl": "~2.8"
     },
+    "require-dev": {
+        "doctrine/doctrine-bundle": "~1.4",
+        "symfony/validator": "~3.3|~4.0",
+        "symfony/yaml": "~3.3|~4.0",
+        "symfony/phpunit-bridge": "~3.3|~4.0",
+        "symfony/security-csrf": "~3.3|~4.0",
+        "symfony/asset": "~3.3|~4.0"
+    },
+    "suggest": {
+        "doctrine/doctrine-bundle": "To use the default dbal configuration"
```

Comment:
well no. If you want to use the default DBAL config, you need to use the DoctrineBundle, as it is the one configuring DBAL. If you have DBAL but not DoctrineBundle, you don't have any default DBAL config